markovflow/kernels/piecewise_stationary.py

```diff
@@ -269,3 +269,20 @@ def state_offsets(self, transition_times: tf.Tensor, time_deltas: tf.Tensor) ->
             ],
             axis=-2,
         )
+
+    def state_means(self, time_points: tf.Tensor) -> tf.Tensor:
```

Comment:
can you run 
'poetry run task format' before commiting
Sorry this is not specified or tested yet.
what this does is enforce some style including max line length,
I think here it would reformat the docstrings

Reply:
I ran the formatter, it didn't change anything here.